conf/app.routes

```diff
@@ -1,9 +1,9 @@
 # microservice specific routes
 
-GET         /assets/*file                                   controllers.Assets.at(path="/public", file)
+GET         /assets/*file                                            controllers.Assets.at(path="/public", file)
 
-GET        /                                                controllers.SoftwareChoicesController.show
-POST       /                                                controllers.SoftwareChoicesController.search
-POST       /ajax                                            controllers.SoftwareChoicesController.ajaxFilterSearch
+GET        /                                                         controllers.SoftwareChoicesController.show
+POST       /                                                         controllers.SoftwareChoicesController.search
+POST       /ajax                                                     controllers.SoftwareChoicesController.ajaxFilterSearch
 
-GET        /accessibility-statement                         controllers.AccessibilityStatementController.show
+GET        /accessibility-statement/:pageUri                         controllers.AccessibilityStatementController.show(pageUri: String)
```

Comment:
I think this needs to be without the argument specified in the URL so it goes through as a query string. Not sure if it will do funny things putting an encoded url at the end of a url.